R/lint.R

```diff
@@ -0,0 +1,144 @@
+# Retrieve the lifecycle status defined in each Rd file
+db_lifecycle <- function(db) {
+  lifecycle_patterns <- paste0("(?:",
```

Comment:
Is this scoped to just the description section? Probably don't want to flag an entire function if only one argument is deprecated.

Reply:
yes, the regexes are only searched in the description section.